spec/models/case_log_spec.rb

```diff
@@ -843,4 +835,73 @@
       expect(case_log.weekly_net_income).to eq(417)
     end
   end
+
+  describe "derived variables" do
+    require "date"
+    let(:organisation) { FactoryBot.create(:organisation) }
+    let!(:case_log) do
+      CaseLog.create({
+        managing_organisation: organisation,
+        owning_organisation: organisation,
+        property_postcode: "M1 1AE",
+        previous_postcode: "M2 2AE",
+        # rubocop:disable Style/DateTime
+        mrcdate: DateTime.new(2021, 5, 4),
```

Comment:
Could you not use Time.new(2021, 5, 4) instead of disabling the Rubocop?

Reply:
That fails the test, it sets the month to 3 instead of 4, I assume because it starts indexing at 0 🤔

Reply:
Weird. But so couldn't you just increment it? Like `Time.new(2021, 5, 5)`?

Reply:
Yeah, I though it'd be confusing to read